cmake/Utils.cmake

```diff
@@ -1,18 +1,12 @@
 #
-# Add MSVC RunTime Flag
+# Add MSVC RunTime Flag, this part is necessary for tests in CI
```

Comment:
Is it only required by CI? or real world case also need it?

CI should just be one instance of real world case in my opinion.

Reply:
Well I added in case few down the road same question is raised.